app/models/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
```

Comment:
- [OPTIONAL] here you don't need the `? true: false` because as soon as `of_age` is true then it will automatically return true even without checking the `@parent_permission`. If your `of_age` is false then it check the `@parent_permission`. Once the both variable evaluated based on the `OR` operator. It will result in `either` or `false` so you don't need to mention the things explicitly. 

```suggestion
    of_age? || @parent_permission
```